app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReviewsController < ApplicationController
+
+  def create
+  end
+
+  def edit
+  end
+
+  def update
+  end
```

Comment:
We aren't creating resources in routes for this. Don't think we'd wan't users to update reviews anyways though?

Reply:
I think if you're the user that left the review you should be able to edit and delete, we can just do some validation in those methods?